netbox/templates/ipam/vlan.html

```diff
@@ -8,7 +8,11 @@
     <div class="col-sm-8 col-md-9">
         <ol class="breadcrumb">
             <li><a href="{% url 'ipam:vlan_list' %}">VLANs</a></li>
-            <li><a href="{% url 'ipam:vlan_list' %}?site={{ vlan.site.slug }}">{{ vlan.site }}</a></li>
+            {% if vlan.site %}
+                <li><a href="{% url 'ipam:vlan_list' %}?site={{ vlan.site.slug }}">{{ vlan.site }}</a></li>
+            {% else %}
+                <li><a href="{% url 'ipam:vlan_list' %}?site_id=0">Global</a></li>
```

Comment:
Let's skip the "global" breadcrumb to be consistent with the way global prefixes are displayed